var/spack/repos/builtin/packages/py-panedr/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPanedr(PythonPackage):
+    """Panedr uses the Pyedr library to read a Gromacs EDR binary energy XDR file and returns
+    its contents as a pandas dataframe"""
+
+    homepage = "https://github.com/MDAnalysis/panedr"
+    pypi = "panedr/panedr-0.7.1.tar.gz"
+
+    maintainers("RMeli")
+
+    version("0.7.1", sha256="64c74863f72d51729ac5cb1e2dbb18747f7137588990c308ef8ca120fbf2ddd4")
+
+    depends_on("py-pyedr@0.7.1", type=("build", "run"), when="@0.7.1")
```

Comment:
Where do you see the version requirement?

Reply:
This is a bit of an assumption with some insider knowledge. Since version `0.6.0`, `panedr` has been split into `pyedr` and `panedr` (which now depends on the former). The two packages live in the same [repository](https://github.com/MDAnalysis/panedr) and are [released together](https://github.com/MDAnalysis/panedr/blob/7268391a60e14b7c065eb1f7c074e14c29e59515/.github/workflows/deploy.yaml#L42-L56), therefore I think it is safer to match their versions.

Reply:
```suggestion
    # PyEDR is released together with PanEDR, therefore versions should match
    depends_on("py-pyedr@0.7.1", type=("build", "run"), when="@0.7.1")
```